File: 448. Find All Numbers Disappeared in an Array/Solution_RE0.java
class Solution {
    public List<Integer> findDisappearedNumbers(int[] nums) {
       List<Integer> tar = new ArrayList<>();
        int[] a = new int[nums.length + 1];
        for (int i = 1; i <=nums.length; i++) {
            a[i] = i;
        }
        for (int i = 0; i < nums.length; i++) {
            a[nums[i]] = 0;
        }
        for (int i = 0; i < a.length; i++) {
            if (a[i] != 0) {
                tar.add(a[i]);
            }
        }
        return tar;
    }
}

File: 448. Find All Numbers Disappeared in an Array/Solution_RE0.py
class Solution:
    def findDisappearedNumbers(self, nums):
        """
        :type nums: List[int]
        :rtype: List[int]
        """
        mark = [0]*(len(nums)+1)
        res =[]
        for num in nums:
            mark[num] = 1
        for i in range(1,len(nums)+1):
            if mark[i]==0:
                res.append(i)
        return res
                
            